### ISSUE TITLE
Issue with NodesInfoResponse

### ISSUE THREAD

**Issue by jgoodall** [olivere/elastic#475]
I believe the `NodesInfoResponse` has an error on this line:
https://github.com/olivere/elastic/blob/4f7c9774c6041d96b9c522cfcbb657a5a49dbbb2/nodes_info.go#L167

Instead of
```
JVM *NodesInfoNodeProcess `json:"jvm"`
```

I think it should be
```
JVM *NodesInfoNodeJVM `json:"jvm"`
```

**Comment by olivere** [comment]
Yep. That looks like a typo.

**Comment by jgoodall** [comment]
See PR #479 for fix

**Comment by olivere** [comment]
Note to self: Keeping this open until backported to v3.

**Comment by olivere** [comment]
Will be fixed in 5.0.29 and 3.0.67 soon. Thanks again.
